### DISCUSSION TITLE
[WPF] ReactiveUI with WPF latest dev build does not load elements

### DISCUSSION THREAD

**Discussion by bstrother1** [unoplatform/uno#9015] [Q&A] [answered]
Current dev build does not work with ReactiveUI and WPF, I'm not sure about other builds only tested with WPF. Element do not load.

WPF: 
    PackageReference Include="HarfBuzzSharp" Version="2.8.2.1-preview.63" 
    PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" 
    PackageReference Include="Microsoft.Extensions.Logging.Console" Version="6.0.0" 
    PackageReference Include="ReactiveUI.Fody" Version="17.1.17" 
    PackageReference Include="ReactiveUI.Uno" Version="17.*" 
    PackageReference Include="Uno.UI.Skia.Wpf" Version="4.4.0-dev.218" 
    PackageReference Include="Uno.UI.RemoteControl" Version="4.4.0-dev.218" Condition="'$(Configuration)'=='Debug'" 

WPF-Host:
    PackageReference Include="Uno.UI.Skia.Wpf" Version="4.4.0-dev.218" 
    PackageReference Include="Uno.UI.RemoteControl" Version="4.4.0-dev.218" Condition="'$(Configuration)'=='Debug'"

[ReactiveUIUnoSample.zip](https://github.com/unoplatform/uno/files/8881064/ReactiveUIUnoSample.zip)

**Comment by jeromelaban** [comment, marked as answer]
You'll need to change the Harfbuzz dependency to be the official one (It will not be needed after https://github.com/unoplatform/uno/pull/9014):
```
<PackageReference Include="SkiaSharp.Harfbuzz" Version="2.80.3" />
```

After that, it works:
![image](https://user-images.githubusercontent.com/5839577/173125338-a562a819-963d-4804-be02-6d5325c162b7.png)
